appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java

```diff
@@ -1264,6 +1266,96 @@ public void DrawLine(int x1, int y1, int x2, int y2) {
     view.invalidate();
   }
 
+  /**
+   * Draws a shape on the canvas, 
+   * point list should be a list contains sub-lists with two number which represents a coordinate
```

Comment:
Consider including an example of what the point list looks like in text form, e.g. ((X1 Y1) (X2 Y2) (X3 Y3)).

Does this logic close the shape automatically, or does (X1 Y1) == (Xn Yn) need to be true? If so, include that in the documentation as well.